### PR TITLE
Adding parsing of "fixes"/"fixed" Keyword and the colon

### LINKED ISSUE #25633: Parsing of labels issue IDs doesn't work with colons and the "fixes" keyword
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

ci

### Describe the bug

I had to manually update the release tags for a PR that used a colon and the "fixes" keyword.

### Version

main

### Expected behavior

The terms `Closes: #123` and `Fixes 123` should be parsed an the issue `123` should be recognized.

### Actual behavior

They are ignored

### How to Reproduce?

https://github.com/keycloak/keycloak/pull/25605 

### Anything else?

I'll prepare a PR

### PR DESCRIPTION
**Pull request by ahus1**
Closes #25633

### CODE REVIEW

File: .github/scripts/pr-find-issues.sh
Comment (jonkoops) [inline .github/scripts/pr-find-issues.sh:1]: Looks like this file now has mixed indentation of either 2 or 4 spaces. Let's keep it consistent with the rest of the script at 2 spaces of indentation.

File: .github/scripts/pr-find-issues-test.sh
Comment (jonkoops) [inline .github/scripts/pr-find-issues-test.sh:1]: Can we change the indentation of this script to 2 spaces like the others?

**Review by jonkoops — DISMISSED** [review]
Just some small nits, but otherwise LGTM

File: .github/scripts/pr-find-issues.sh
Comment (ahus1) [reply .github/scripts/pr-find-issues.sh:1]: I've updated it to 4 spaces as it was using mostly 4 spaces before my change.

File: .github/scripts/pr-find-issues-test.sh
Comment (ahus1) [reply .github/scripts/pr-find-issues-test.sh:1]: see above - I've updated it to 4 spaces as it was using mostly 4 spaces before my change.

**Review by jonkoops — APPROVED** [review]

File: .github/scripts/pr-find-issues-test.sh
Comment (mhajas) [inline .github/scripts/pr-find-issues-test.sh:1]: Where do we run this test? Is this something to just run locally when doing some changes or we should run it in CI?

Comment (mhajas) [reply .github/scripts/pr-find-issues-test.sh:1]: Would be probably better to run it in CI but it is probably not worth the effort.

**Review by mhajas — APPROVED** [review]
thank you @ahus1

**Pull request merged**